docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM python:alpine as production
+
+# Configuration params
+ARG INVENTREE_REPO="https://github.com/inventree/InvenTree.git"
+ARG INVENTREE_VERSION="master"
+ARG INVENTREE_HOME="/home/inventree"
+
+ENV PYTHONUNBUFFERED 1
+
+# InvenTree paths
+ENV INVENTREE_SRC_DIR="$INVENTREE_HOME/src"
+ENV INVENTREE_STATIC_ROOT="$INVENTREE_HOME/static"
+ENV INVENTREE_MEDIA_ROOT="$INVENTREE_HOME/media"
+ENV INVENTREE_LOG_DIR="$INVENTREE_HOME/log"
+ENV INVENTREE_BACKUP_DIR="$INVENTREE_HOME/backup"
+ENV INVENTREE_VENV="$INVENTREE_HOME/env"
+
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.build-date=$DATE \
+      org.label-schema.vendor="inventree" \
+      org.label-schema.name="inventree/inventree" \
+      org.label-schema.url="https://hub.docker.com/r/inventree/inventree-docker" \
+      org.label-schema.version=$INVENTREE_VERSION \
+      org.label-schema.vcs-url=$INVENTREE_REPO \
+      org.label-schema.vcs-branch=$BRANCH \
+      org.label-schema.vcs-ref=$COMMIT
+
+RUN echo "Installing InvenTree '${INVENTREE_VERSION}' from ${INVENTREE_REPO}"
+
+# Create user account
+RUN addgroup -S inventreegroup && adduser -S inventree -G inventreegroup
+WORKDIR /home/inventree
+
+# Install required system packages
+RUN apk add --no-cache git make bash \
+    gcc libgcc g++ libstdc++ \
+    libjpeg-turbo libjpeg-turbo-dev jpeg jpeg-dev \
+    libffi libffi-dev \
+    zlib zlib-dev
+RUN apk add --no-cache cairo cairo-dev pango pango-dev
+RUN apk add --no-cache fontconfig ttf-droid ttf-liberation ttf-dejavu ttf-opensans ttf-ubuntu-font-family font-croscore font-noto
+RUN apk add --no-cache python3
+RUN apk add --no-cache postgresql-contrib postgresql-dev libpq
+RUN apk add --no-cache mariadb-connector-c mariadb-dev
+
+# Clone source code
+RUN git clone --branch $INVENTREE_VERSION --depth 1 ${INVENTREE_REPO} ${INVENTREE_SRC_DIR}
+
+# Install required PIP packages
+RUN python -m venv $INVENTREE_VENV && pip install --upgrade pip setuptools wheel
```

Comment:
So playing with the Dockerfile tonight, this doesn't install anything in to $INVENTREE_VENV... my fault here.

```
+RUN python -m venv $INVENTREE_VENV 
+RUN $INVENTREE_VENV/bin/python3 -m pip install --upgrade pip setuptools wheel
+RUN $INVENTREE_VENV/bin/pip install --no-cache-dir -U invoke
+RUN $INVENTREE_VENV/bin/pip install --no-cache-dir -U psycopg2 mysqlclient pgcli mariadb gunicorn
```

Reply:
Fixed in latest